NBitcoin/UInt2561.cs

```diff
@@ -662,14 +667,19 @@ public uint160(byte[] vch)
 		public override bool Equals(object obj)
 		{
 			var item = obj as uint160;
-			if(item == null)
+			return Equals(item);
+		}
+
+		public bool Equals(uint160 other)
+		{
+			if (other == null)

```

Comment:
can you do `other is null` instead, so we are sure no operator overload is used?